pin/pin.go

```diff
@@ -26,8 +25,9 @@ var emptyKey *cid.Cid
 func init() {
 	e, err := cid.Decode("QmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n")
 	if err != nil {
-		log.Error("failed to decode empty key constant")
-		os.Exit(1)
+		msg := "failed to decode empty key constant"
+		log.Error(msg)
+		panic(msg)
```

Comment:
I didn't want to exit over there as it would change the coverage, but as it shouldn't happen here I will switch back to `Exit`.